ribasim_qgis/widgets/dataset_widget.py

```diff
@@ -50,10 +50,10 @@ def __init__(self, parent: QWidget | None):
         self.setSizePolicy(QSizePolicy.Minimum, QSizePolicy.Preferred)
         self.setHeaderLabels([""])
         self.setHeaderHidden(False)
+        self.setColumnCount(1)
         header = self.header()
```

Comment:
Do we still need any header logic? There's only a single column, so there's no sections to speak of. The resizing behavior only matters for multiple columns too.